lib/parser/parse-tree.cc

```diff
@@ -80,9 +80,26 @@ Expr::Expr(Designator &&x)
 Expr::Expr(FunctionReference &&x)
   : u{common::Indirection<FunctionReference>::Make(std::move(x))} {}
 
+const std::optional<LoopControl> &DoConstruct::GetLoopControl() const {
+  NonLabelDoStmt const &doStmt{
```

Comment:
Everywhere else we put `const` before the type so we should stick with that style (or change it everywhere).

Reply:
I'm surprised that this doesn't cause a syntax error.  Thanks for catching it.